### ISSUE TITLE
Modalities column in the tables

### ISSUE THREAD

**Issue by sunnyHelen** [Lucas-rbnt/bmmae#4]
Hi Author Team,

Congratulations on the great work! And thanks for sharing your code and checkpoints.

I had a quick question regarding the results table. Specifically, for the column indicating modalities, does this refer to the pretraining modality, the finetuning modality, or the test modality?

**Comment by Lucas-rbnt** [comment]
Hi, thank you for your interest. 

The column indicating the modalities refers to the fine-tuning modalities. Pre-training is done jointly on all modalities; afterwards, the model can handle any subset at inference or fine-tuning.
We reported results with fixed modality sets to ensure fair apples-to-apples comparisons with baselines that are restricted to these scenarios.  The architecture itself supports mixed-modality usage: you can test, infer, fine-tune or even pre-train with arbitrary combinations, including datasets where samples may have different available modalities.

Out of curiosity, why are you interested in this repo?

Have a nice day!
Lucas

**Comment by sunnyHelen** [comment]
Thanks for your quick reply. I see. Did you try pretrain the model with a single modality, for example, t1ce?
I'm also involved in the downstream tasks of Glioma classification and segmentation. That's why I feel your work is very valuable.

**Comment by Lucas-rbnt** [comment]
Hi!

Pre-training on a single modality corresponds to the MAE column in our work. To train a standard MAE, you can either use the MONAI implementation (which I actually wrote, ahah) or adapt the regular 2D MAE. MONAI also comes with a less restrictive license.

If you have T1ce data (not from BraTS), one option is to take the pre-trained weights from this repo and fine-tune them on your own dataset. You could even run another round of pre-training on top of these weights before fine-tuning depending on your labels.

Lucas.

**Comment by sunnyHelen** [comment]
Thanks a lot for your kind insight, Lucas! I'll definitely try.

**Comment by Lucas-rbnt** [comment]
No problem, feel free to reach back if you have any questions!
